src/components/Integration.tsx

```diff
@@ -511,7 +512,7 @@ export function IntegrationPanel(props: IProps) {
   }, [busy, paratext_projects, paratext_projectsStatus]);
 
   useEffect(() => {
-    findConnectedProject();
+    if (project) findConnectedProject();
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [projectintegrations]);
```

Comment:
Should we condition on project change here too?

Reply:
I don't think so... this would only change if they choose a paratext project, in which case the project should have been long ago set.

Reply:
but I guess I'm not sure how the project is ever null to begin with, because by the time we get here from the transcriber window, we should have known project for sure.  So it shouldn't hurt to put it in, because we aren't expecting project to change ever.